cmd/keytransparency-sequencer/main.go

```diff
@@ -183,21 +167,20 @@ func main() {
 	grpc_prometheus.Register(grpcServer)
 	grpc_prometheus.EnableHandlingTimeHistogram()
 
-	glog.Infof("Signer starting")
-
 	// Run servers
-	go func() { glog.Error(serverutil.ServeHTTPMetrics(*metricsAddr, serverutil.Readyz(sqldb))) }()
-	go serveHTTPGateway(ctx, lis, dopts, grpcServer,
-		pb.RegisterKeyTransparencyAdminHandlerFromEndpoint,
-	)
-	runSequencer(ctx, conn, directoryStorage)
+	g, gctx := errgroup.WithContext(ctx)
+	g.Go(func() error { return serverutil.ServeHTTPMetrics(*metricsAddr, serverutil.Readyz(sqldb)) })
+	g.Go(func() error {
+		return serverutil.ServeHTTPAPIAndGRPC(gctx, lis, grpcServer, conn,
+			pb.RegisterKeyTransparencyAdminHandler)
+	})
+	g.Go(func() error { return runSequencer(gctx, conn, directoryStorage) })
```

Comment:
what's the impact (if any) of running runSequencer in a goroutine as opposed to inline like before?

Reply:
The impact is that this function now returns. I suppose it doesn't need to be launched with g.Go does it?